Makefile

```diff
@@ -147,3 +147,15 @@ local_env_create: ## Create a virtualized lab on your local machine. Nested virt
 	else \
 		ansible-playbook -i localhost, -c local dev-local-env.yml ${LOCAL_ENV_OPTS}; \
 	fi
+
+##@ Generate documentation
+VIRTENV_FOLDER_NAME = virtdocs
+.PHONY: docs
+docs:
+	cd docs && \
+	virtualenv $(VIRTENV_FOLDER_NAME) && \
+	. virtdocs/bin/activate && \
+	pip install -r doc-requirements.txt && \
```

Comment:
that would be pip install -r ../doc-requirements.txt then, I think?

Reply:
doc-requirements.txt is under docs